shared/src/commonMain/kotlin/com/rootstrap/flowforms/shared/FormModel.kt

```diff
@@ -0,0 +1,59 @@
+package com.rootstrap.flowforms.shared
+
+import com.rootstrap.flowforms.core.common.StatusCodes
+import com.rootstrap.flowforms.core.dsl.flowForm
+import com.rootstrap.flowforms.core.field.FlowField
+import com.rootstrap.flowforms.core.form.FlowForm
+import com.rootstrap.flowforms.core.validation.BasicEmailFormat
+import com.rootstrap.flowforms.core.validation.Match
+import com.rootstrap.flowforms.core.validation.MinLength
+import com.rootstrap.flowforms.core.validation.Required
+import com.rootstrap.flowforms.core.validation.RequiredTrue
+import com.rootstrap.flowforms.core.validation.on
+import kotlinx.coroutines.Dispatchers
+
+class StatusCode {
+    var code = StatusCodes
+}
```

Comment:
I think this class is not needed anymore because `StatusCodes` is working correctly now